build.psm1

```diff
@@ -446,6 +446,32 @@ Fix steps:
         Pop-Location
     }
 
+    # publish powershell.config.json
+    # only Windows currently has a default config file that needs to be merged if Preview
+    $version = git --git-dir="$PSSCriptRoot/.git" describe
```

Comment:
I think we should use `$ReleaseTag` instead. `git --git-dir describe` uses the latest tag, which will always be a preview version until after we release the GA version (tag is pushed after all build validations are done).

Reply:
Will need to fallback to the git command as `$ReleaseTag` isn't defined, but passed in.

Reply:
When `$ReleaseTag` is not specified, it would be a self-build. For a release build, this parameter will always be specified.

Reply:
> it would be a self-build

Not accurate. It could also be a CI build I think.